src/NLog.Web.AspNetCore/LayoutRenderers/AspNetTlsTokenBindingLayoutRenderer.cs

```diff
@@ -0,0 +1,85 @@
+﻿#if ASP_NET_CORE3
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Http.Features;
+using NLog.Config;
+using NLog.LayoutRenderers;
+using NLog.Web.Enums;
+using NLog.Web.Internal;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET TLS Token Bindings for Provider and Referrer
+    ///
+    /// TLS token bindings help mitigate the risk of impersonation by an attacker in the event
+    /// an authenticated client's bearer tokens are somehow exfiltrated from the client's machine.
+    /// See https://datatracker.ietf.org/doc/draft-popov-token-binding/ for more information.
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-tls-token-binding:Property=Provider:Format=Base64}
+    /// ${aspnet-tls-token-binding:Property=Referrer:Format=Hex} for hexadecimal separated by the dash character
+    /// </remarks>
+    [LayoutRenderer("aspnet-tls-token-binding")]
+    public class AspNetTlsTokenBindingLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Could these 2 extra TLS properties, not be added to the existing TLS-Property-LayoutRenderer ?